builder/dispatchers.go

```diff
@@ -52,7 +60,8 @@ func env(b *Builder, args []string, attributes map[string]bool, original string)
 	for j := 0; j < len(args); j++ {
 		// name  ==> args[j]
 		// value ==> args[j+1]
-		newVar := args[j] + "=" + args[j+1] + ""
+		newValue := b.replaceEnv(args[j+1])
+		newVar := args[j] + "=" + newValue + ""
```

Comment:
Been looking at that on-n-off all day trying to remember what the heck I was doing in that previous PR but I have no idea  :-)  I've removed it.